File: vortexdbg-api/src/main/java/com/vortexdbg/debugger/gdb/DetachCommand.java
package com.vortexdbg.debugger.gdb;

import com.vortexdbg.Emulator;

class DetachCommand implements GdbStubCommand {

    @Override
    public boolean processCommand(Emulator<?> emulator, GdbStub stub, String command) {
        stub.makePacketAndSend("OK");
        stub.detachServer();
        return true;
    }

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/debugger/gdb/DetachCommand.kt
package com.vortexdbg.debugger.gdb

import com.vortexdbg.Emulator

internal class DetachCommand : GdbStubCommand {

    override fun processCommand(emulator: Emulator<*>, stub: GdbStub, command: String): Boolean {
        stub.makePacketAndSend("OK")
        stub.detachServer()
        return true
    }

}
